sdk/cloudmachine/Azure.Projects.Provisioning/src/Features/AIFoundry/AIProjectFeature.cs

```diff
@@ -72,30 +72,27 @@ private  void EmitConnections(ICollection<ClientConnection> connections, string
         /// Emit any necessary resources for provisioning (currently no-op).
         /// </summary>
         /// <param name="infrastructure">The ProjectInfrastructure context.</param>
-        /// <returns>A placeholder or no-op resource, as provisioning is out-of-band for now.</returns>
-        protected override ProvisionableResource EmitResources(ProjectInfrastructure infrastructure)
+        protected internal override void EmitConstructs(ProjectInfrastructure infrastructure)
         {
             var cmId = infrastructure.ProjectId;
-            AIFoundryHubCdk hub = new($"{cmId}_hub", $"{cmId}_hub");
-            AIFoundryProjectCdk project = new($"{cmId}_project", $"{cmId}_project", hub);
+            AIFoundryHubCdk hub = new($"ai_hub", $"{cmId}_hub");
```

Comment:
maybe for later, but should we handle being able to create multiple of these where the name doesn't conflict?

Reply:
Yeah, I was planning to do it, but it's a complex feature. Some resources are fundamentally singletons and some are not, and some a in the middle (e.g. you can have one app config, unless you guy a premium sku)